_i18n/en/resources/developer-guides/daemon-rpc.md

```diff
@@ -563,270 +783,1636 @@ $ curl -X POST http://127.0.0.1:18081/json_rpc -d '{"jsonrpc":"2.0","id":"0","me
+
+Get the transaction pool statistics.
+
+Alias: *None*.
+
+Inputs: *None*.
+
+Outputs:
+
+* *pool_stats* - Structure as follows:
+  * *bytes_max* - unsigned int; Max transaction size in pool
+  * *bytes_med* - unsigned int; Median transaction size in pool
+  * *bytes_min* - unsigned int; Min transaction size in pool
+  * *bytes_total* - unsigned int; total size of all transactions in pool
+  * *histo* - structure *txpool_histo* as follows:
+    * *txs* - unsigned int; number of transactions
+    * *bytes* - unsigned int; size in bytes.
+  * *histo_98pc* unsigned int; the time 98% of txes are "younger" than
+  * *num_10m* unsigned int; number of transactions in pool for more than 10 minutes
+  * *num_double_spends* unsigned int; number of double spend transactions
+  * *num_failing* unsigned int; number of failing transactions
+  * *num_not_relayed* unsigned int; number of non-relayed transactions
+  * *oldest* unsigned int; time of the oldest transaction in the pool
+  * *txs_total* unsigned int; total number of transactions.
+* *status* - string; General RPC error code. "OK" means everything looks good.
+* *untrusted* - boolean;
+
+Example:
+
+```
+$ curl -X POST http://127.0.0.1:18081/get_transaction_pool_stats -H 'Content-Type: application/json'
+
+{
+  "pool_stats": {
+    "bytes_max": 47222,
+    "bytes_med": 13290,
+    "bytes_min": 13092,
+    "bytes_total": 449511,
+    "fee_total": 289715320000,
+    "histo": "\t▒'▒5▒4▒\/▒▒▒$3",
+    "histo_98pc": 0,
+    "num_10m": 18,
+    "num_double_spends": 1,
+    "num_failing": 17,
+    "num_not_relayed": 0,
+    "oldest": 1525457001,
+    "txs_total": 26
+  },
+  "status": "OK",
+  "untrusted": false
+}
+```
+
+
+### **/stop_daemon**
+
+Send a command to the daemon to safely disconnect and shut down.
+
+Alias: *None*.
+
+Inputs: *None*.
+
+Outputs:
+
+* *status* - string; General RPC error code. "OK" means everything looks good.
+
+Example:
+
+```
+$ curl -X POST http://127.0.0.1:18081/stop_daemon -H 'Content-Type: application/json'
+
+{
+  "status": "OK"
+}
+```
+
+
+### **/get_info (legacy)**
+
+This method is a convenient backward support and should not be used anymore. See [get_info](#get_info) JSON RPC for details.
+
+Alias:
+
+* */getinfo*
+* *get_info*
+
+
+### **/get_limit**
+
+Get daemon bandwidth limits.
+
+Alias: *None*.
+
+Inputs: *None*.
+
+Outputs:
+
+* *limit_down* - unsigned int; Download limit in kBytes
+* *limit_up* - unsigned int; Upload limit in kBytes
+* *status* - string; General RPC error code. "OK" means everything looks good.
+* *untrusted* - boolean;
+
+Example:
+
+```
+$ curl -X POST http://127.0.0.1:18081/get_limit -H 'Content-Type: application/json'
+
+{
+  "limit_down": 8192,
+  "limit_up": 128,
+  "status": "OK",
+  "untrusted": false
+}
+```
+
+
+### **/set_limit**
+
+Set daemon bandwidth limits.
+
+Alias:
+
+* *limit_down* - unsigned int; Optional, download limit in kBytes
+* *limit_up* - unsigned int; Optional, upload limit in kBytes
```

Comment:
signed ints, -1 means set reset to default, 0 means don't change.

Reply:
ok

Reply:
Btw, it has nothing to do in Alias section. It should be in Inputs.